### PR TITLE
Add: Badge component

### PR DESCRIPTION
**Pull request by hussiantabha**
Bade with Icon and text

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This looks fine, you can merge.

File: components/badges.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+:root{
+    --primary: blueviolet;
+    --white: #fff;
+}
```
Comment (arp99) [inline components/badges.css]: Keep :root in the main css file, do not give :root in individual files

File: components/badges.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.container-badge{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 1rem 0;
+}
+.badges{
+    margin-right: 2rem;
+    position: relative;
+}
```
Comment (arp99) [inline components/badges.css]: What does the class .badges  do ?

**Review by arp99 — COMMENTED** [review]
Added some suggestions

File: components/badges.css (hunk 2 above)
Comment (hussiantabha) [reply components/badges.css]: It's the parent container which hold the icon with the badge

**Review by arp99 — APPROVED** [review]
Looks good

**Pull request merged**